### PR TITLE
Risk input preparation tool improvements 

### PR DESCRIPTION
**Pull request by bwyss**
This PR includes many improvements and new features that can be found in the attached PDF (completed features are indicated by being crossed out)

Most notable features:
Revers the x / y axis of the fragility discrete tables
Add intensity measure as a column for the fragility discrete tables
Add intensity measure as a column for the vulnerability discrete tables

[Risk Input Preparation Toolkit - Google Docs SnapShot.pdf](https://github.com/gem/oq-platform/files/20914/Risk.Input.Preparation.Toolkit.-.Google.Docs.SnapShot.pdf)

Note: Addition work needs to be done on the vulnerability tables, this refactor is complete for Vulnerability discrete tables, but is not complete for probability mass function tables. This work in progress has been moved into the "risk-ipt-new-vuln-tables" branch.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -9,3 +9,15 @@
...
+  * Revers the x / y axis of the fragility discrete tables.
+
+  * Add intensity measure as a column for the fragility discrete tables.
+
+  * Add intensity measure as a column for the vulnerability discrete tables.
+
+  * Vulnerability discrete table header is generated dynamically when the user selects an IML from a provided selection menu.
+
+  * If the selected IML for a discrete vulnerability function is "SA" then the user is prompted to provide a value for the SA period before the table is generated.
+
```
Comment (nastasi-oq) [inline CHANGELOG]: Sintax is not fully compliant our format:
  * no white lines between items of the same developer
  * lines length no more than 80 characters
Here a full description: https://github.com/gem/oq-engine/blob/master/doc/engine/Developers-Notes.md#guidelines-for-debianchangelog-file

File: CHANGELOG [hunk 2]
```diff
@@ -9,3 +9,11 @@
   * vulnerability: replace limit states lowercased names with themselves where
     all [^a-z0-9] chars are substituted with '_' and contiguous '_' are
     collapsed into one
+
+  [Ben Wyss]
+  * Revers the x / y axis of the fragility discrete tables.
+  * Add intensity measure as a column for the fragility discrete tables.
+  * Add intensity measure as a column for the vulnerability discrete tables.
+  * Vulnerability discrete table header is generated dynamically when the user selects an IML from a provided selection menu.
```
Comment (nastasi-oq) [inline CHANGELOG]: not more than 80 characters per line

File: openquakeplatform/openquakeplatform/static/css/absolute_center_spinner.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    -moz-transform: rotate(360deg);
+    -ms-transform: rotate(360deg);
+    -o-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
```
Comment (nastasi-oq) [inline openquakeplatform/openquakeplatform/static/css/absolute_center_spinner.css]: New line is missing.

**Comment by daniviga** [comment]
I don't know if it's the right place to signal that during the risk workshop we had some troubles copy and pasting cells because when "ctrl" is hit on the table in the exposure tab (i.e. doing ctrl-c, ctrl-v) a new line is added to the table just below the selected cell.

**Comment by bwyss** [comment]
The issue with copying and pasting cells was only present in Ubuntu/FireFox. This issue has been resolved in a newer version of the handsontable library. I included in this branch an upgrade of that lib which resolves the issue.

**Comment by daniviga** [comment]
Ben, the issue I'm referring to is reproducible with this branch, running both Chrome and Firefox on Fedora Linux, and on the VM which uses Lubuntu 14.04 and is different from the one you were explaining us last week (paste in a single cell instead of in the right cells). After lunch I will test it also on Windows.

**Comment by daniviga** [comment]
I can confirm the same issue on Windows (sorry, only XP) + Firefox: https://youtu.be/R6QqygLbzRE

**Comment by bwyss** [comment]
I misunderstood the issue. Please find the new commits that should resolve the issue.

**Comment by daniviga** [comment]
Thanks Ben! Yes, that solved the issue with `ctrl`, at least on Linux. I'm going to test it on Mac and Win and I'll report you.

BTW I still get more than one new line after playing with the drop down menus.

**Comment by daniviga** [comment]
I confirm that `ctrl + shift + n` works well in Win and Mac

**Comment by daniviga** [comment]
The issue about more rows added by `ctrl + shift +n` is non blocking for me. I've opened an issue https://github.com/gem/oq-platform/issues/503

**Comment by nastasi-oq** [comment]
LGTM

**Comment by daniviga** [comment]
LGTM for me too (c/p tested on F22/F23 + Chrome + Firefox from LibreOffice and Windows + FF from Excel 2010)

**Pull request merged**